src/interpreter/internal.rs

```diff
@@ -224,6 +224,8 @@ mod tests {
             vec![vec![].into()],
         );
 
+        let tx = Interpreter::<()>::check_transaction(tx, 0).unwrap();
```

Comment:
Consider using `expect` with a message that clarifies what would be the expected behavior here